### PR TITLE
feat: DevelopmentAPI Sidebar

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
We want to customize the sidebar of the OpenAPI docs and show something similar to the [Stripe docs](https://stripe.com/docs).
If a user wants to exit the OpenAPI Reference then they can use the breadcrumbs navigation.

## UI
Screen Shot 2020-09-08 at 11 45 16 AM
Screen Shot 2020-09-08 at 11 45 27 AM

### CODE REVIEW

File: src/components/developmentApiSidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  let endpoints = tree[0].children.reduce((acc: EntityTree[], curr: EntityTree) => {
+    if (curr.children.length > 1) {
+      acc.push(curr);
+    }
+    return acc;
+  }, []);
+
+  const location = useLocation();
+  const isActive = path =>
```
Comment (mgaeta) [inline src/components/developmentApiSidebar.tsx:36]: Is `.startsWith` available?

File: src/components/developmentApiSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+export default () => {
+  const data = useStaticQuery(query);
+  const tree = toTree(data.allSitePage.nodes.filter(n => !!n.context));
+
+  let endpoints = tree[0].children.reduce((acc: EntityTree[], curr: EntityTree) => {
+    if (curr.children.length > 1) {
+      acc.push(curr);
+    }
+    return acc;
+  }, []);
```
Comment (mgaeta) [inline src/components/developmentApiSidebar.tsx:33]: I think you should use a `.filter`: 
```
const endpoints = tree[0].children.filter(child => child.children.length > 1);
```

**Review by mgaeta — APPROVED** [review]
Cool 👍

**Comment by manuzope** [comment]
Should https://docs.sentry.io/development-api/ show the new sidebar?

**Pull request merged**